src/components/send/SendHeader.js

```diff
@@ -218,10 +210,6 @@ export default function SendHeader({
 
   return (
     <Fragment>
-      <SheetHandleFixedToTop />
```

Comment:
we sure?

Reply:
yeah pretty sure, moved it into `SendSheet` bc the `SendHeader` isn't displayed if there are no assets to send but we still want to see the sheet handle/title